examples/plugin/module/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+set(classes
+  vtkF3DExampleReader
+  )
+
+vtk_module_add_module(f3d::VTKExtensionsExampleReader
```

Comment:
not a VTKExtensions imo

Reply:
Why?

Reply:
a VTKExtensions module is conceptually an VTK modules extending another software (F3D). This is just a VTK module for a plugin. also should not be part of f3d:: either.

I would just call it ExampleReader tbh.

Reply:
It is a VTK module extending the f3d plugin reader capabilities. I agree about the namespace though.

Reply:
as you see fit

Reply:
looks like you kept the f3d:: namespace, is that intended ?

Reply:
changed `f3d::` to `example::`